### PR TITLE
Added support for Red Dead Redemption 2 - RedM

### PR DESCRIPTION
**Pull request by ZixeSea**
RedM and FiveM are the same including the default port 30120

### CODE REVIEW

**Comment by CosminPerRam** [comment]
Thanks for adding the support for RedM.

Tested it on some random servers and everything seems fine, can you rebase please? 
Also, you could add a line to the [CHANGELOD.md](https://github.com/gamedig/node-gamedig/blob/master/CHANGELOG.md) file indicating the support has been added.

**Comment by ZixeSea** [comment]
Not sure why I did the pull in 2 steps but it should be good now

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 
 ### To Be Released...
 * Sons Of The Forest - Added support
+* Red Dead Redemption 2 - Added support
```
Comment (CosminPerRam) [inline CHANGELOG.md:4]: Nitpicking a bit, make sure this matches the other files game name (missing the `- RedM (2018)` part)

Comment (ZixeSea) [reply CHANGELOG.md:4]: Valid

**Pull request merged**